SplashScreen/app/src/main/java/com/example/android/splashscreen/MainActivity.kt

```diff
@@ -15,19 +17,26 @@ import com.example.android.splashscreen.databinding.MainActivityBinding
 
 /**
  * Shows the app content that is commonly used in all of [DefaultActivity], [AnimatedActivity], and
- * [CustomActivity]. This also handles the custom dark mode.
+ * [CustomActivity]. This also handles the custom dark mode on API level 31 and above.
  */
 abstract class MainActivity : AppCompatActivity() {
 
     private val viewModel: MainViewModel by viewModels()
 
+    protected lateinit var installedSplashScreen: SplashScreen
```

Comment:
I'd prefer to call this just splashScreen. There in no reason to qualify the splash screen name since there can be only one.

Reply:
Done.
I thought that it would conflict with the platform getSplashScreen, but it seems to be fine.